src/Microsoft.NetCore.Analyzers/Core/Security/DoNotUseInsecureDeserializerLosFormatter.cs

```diff
@@ -15,14 +15,13 @@ namespace Microsoft.NetCore.Analyzers.Security
     [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
     public class DoNotUseInsecureDeserializerLosFormatter : DoNotUseInsecureDeserializerMethodsBase
     {
-        // TODO paulming: Help link URLs.
         internal static DiagnosticDescriptor RealMethodUsedDescriptor =
             SecurityHelpers.CreateDiagnosticDescriptor(
                 "CA2305",
                 nameof(MicrosoftNetCoreSecurityResources.LosFormatterMethodUsedTitle),
                 nameof(MicrosoftNetCoreSecurityResources.LosFormatterMethodUsedMessage),
                 isEnabledByDefault: false,
-                helpLinkUri: null);
+                helpLinkUri: "https://docs.microsoft.com/visualstudio/code-quality/ca2305-do-not-use-insecure-deserializer-losformatter");
```

Comment:
Would you be able run CIBuild.cmd locally to even update the .md file? I am going to spend some time today to try to enable executing this tool on all local builds.

Reply:
Not working for me; will debug with you later.